streams-tests/jvm-native/src/test/scala/zio/stream/ZStreamPlatformSpecific2Spec.scala

```diff
@@ -0,0 +1,117 @@
+package zio.stream
+
+import zio._
+import zio.test.Assertion._
+import zio.test._
+
+import java.io._
+import java.nio.charset.StandardCharsets
+import java.nio.file.{Files, NoSuchFileException, Paths}
+
+object ZStreamPlatformSpecific2Spec extends ZIOBaseSpec {
+
+  def spec = suite("ZStream JVM - Native")(
+    suite("Constructors")(
+      suite("fromFile")(
+        test("reads from an existing file") {
+          val data = (0 to 100).mkString
+
+          ZIO.acquireReleaseWith {
+            ZIO.attempt(Files.createTempFile("stream", "fromFile"))
+          } { path =>
+            ZIO.attempt(Files.delete(path)).orDie
+          } { path =>
+            ZIO.attempt(Files.write(path, data.getBytes(StandardCharsets.UTF_8))) *>
+              assertZIO(
+                ZStream
+                  .fromPath(path, 24)
+                  .via(ZPipeline.utf8Decode)
+                  .mkString
+              )(equalTo(data))
+          }
+        },
+        test("fails on a nonexistent file") {
+          assertZIO(ZStream.fromPath(Paths.get("nonexistent"), 24).runDrain.exit)(
+            fails(isSubtype[NoSuchFileException](anything))
+          )
+        }
+      ),
+      suite("from")(
+        suite("java.util.stream.Stream")(
+          test("JavaStream") {
+            trait A
+            trait JavaStreamLike[A] extends java.util.stream.Stream[A]
+            lazy val javaStream: JavaStreamLike[A]        = ???
+            lazy val actual                               = ZStream.from(javaStream)
+            lazy val expected: ZStream[Any, Throwable, A] = actual
+            lazy val _                                    = expected
+            assertCompletes
+          },
+          // test("JavaStreamScoped") {
```

Comment:
Should this remain?

Reply:
Oh I see this existed in the code you migrated this from.

Reply:
Probably not, but I was unsure of its original purpose...